### ISSUE TITLE
not all figure titles are using class=figureTitle

### ISSUE THREAD

**Issue by JohnMoehrke** [IHE/publications#22]

**Comment by JohnMoehrke** [comment]
Recommendation for Figures. This will keep the image and the title together 

```
          <div class="figure">
            <img src="media/Figure_16.1-1.png" alt="" />
            <p class="figureTitle">Figure 16.1-1: XDM Actor Diagram </p>
          </div>

```

**Comment by JohnMoehrke** [comment]
I think this one is done and should be closed

**Comment by JohnMoehrke** [comment]
@vassilpeytchev is this one done and should be closed?

**Comment by JohnMoehrke** [comment]
is this solved with #81

**Comment by lukeaduncan** [comment]
Solved by #81
